docs/make.jl

```diff
@@ -24,7 +24,8 @@ examples = [
     "Simple column model" => "column",
     "Baroclinic instability" => "eady",
     "Data forced column model" => "data_forced",
-    "Model with particles (kelp) interacting with the biogeochemistry" => "kelp"
+    "Model with particles (kelp) interacting with the biogeochemistry" => "kelp",
+    #"Data assimilation" => "data_assimilaiton"
```

Comment:
if it's commented out why have it?

Reply:
I want the other changes to build successfully first then I'll uncomment so we can see how much extra time it adds.